data/json/items/book/fabrication.json

```
@@ -390,8 +390,8 @@
     "max_level": 2,
     "intelligence": 5,
     "proficiencies": [
-      { "proficiency": "prof_carpentry_basic", "time_factor": 0.4, "fail_factor": 0.8, "include_prereqs": true },
-      { "proficiency": "prof_plumbing", "time_factor": 0.5, "fail_factor": 0.8, "include_prereqs": true }
+      { "proficiency": "prof_carpentry_basic", "time_factor": 0.2, "fail_factor": 0.2, "include_prereqs": true },
+      { "proficiency": "prof_plumbing", "time_factor": 0.2, "fail_factor": 0.3, "include_prereqs": true }
```

```suggestion
      { "proficiency": "prof_carpentry_basic", "time_factor": 0.2, "fail_factor": 0.3, "include_prereqs": true },
      { "proficiency": "prof_plumbing", "time_factor": 0.2, "fail_factor": 0.2, "include_prereqs": true }
```
Given that it's described as for the "novice carpenter".